src/main/java/com/github/krzysiekjodlowski/ox/Symbol.java

```diff
@@ -0,0 +1,30 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Represents each player on Board.
+ * @author Krzysztof Jodlowski
+ */
+enum Symbol {
+    NAUGHT("O"), CROSS("X");
```

Comment:
NAUGHT and CROSS, nice names :) I usually see just O and X, so it was a nice surprise

Reply:
Found it on requirements;)